test/helpers/serviceBuilders.js

```diff
@@ -23,6 +23,12 @@ export const defaultProviderConfig = {
 };
 
 export const websocketProviderConfig = {
+  accounts: {
+    default: {
+      key: '0x474beb999fed1b3af2ea048f963833c686a0fba05f5724cb6417cf3b8ee9697e',
+      type: 'privateKey'
+    }
+  },
```

Comment:
Testing with ganache typically uses the predefined unlocked accounts stored in `testAccounts.json`. However, these are usually not explicitly instantiated as above as the accounts service will make an rpc call (`eth_accounts`), to get the first of the accounts directly from the blockchain. 
For whatever reason, this will cause subscriptions not to work. When the privatekey for the same account is explicitly stated in the configuration as above, it does work.

Reply:
I wonder if this is related to the issue with accepting log events that don't have the correct address data?

Reply:
Yes it is